### PR TITLE
fix: test example

### PR DESCRIPTION
**Pull request by WuXuan374**
## Description
- related to issue #500 
- suggested @hugochan as reviewer

## Descriptions
- bug fix in an example inside test/

### CODE REVIEW

**Comment by hugochan** [comment]
@WuXuan374 thanks for contributing to the Graph4NLP library! I got the error `ModuleNotFoundError: No module named 'graph4nlp.pytorch.modules.graph_construction.embedding_construction` when running ` python -m graph4nlp.pytorch.test.graph_construction.run_batch_graph_embedding_construction`. It seems you also need to update the import path. Can you please take a look and fix this? Thanks!

**Review by hugochan — CHANGES_REQUESTED** [review]
@WuXuan374 thanks for contributing to the Graph4NLP library! I got the error ModuleNotFoundError: No module named 'graph4nlp.pytorch.modules.graph_construction.embedding_construction when running python -m graph4nlp.pytorch.test.graph_construction.run_batch_graph_embedding_construction. It seems you also need to update the import path. Can you please take a look and fix this? Thanks!

**Comment by WuXuan374** [comment]
@hugochan thanks for your reply. Here's the description for my second commit:

1.  fix import path. 
2. I mock a vocab_util.py with only 2 modifications compare to '...modules.utils.vocab_utils' (You can easily search them with comment "Modification"). The reason is in our test example, 'str' has no extract() method specified (which would be defined in a class like Text2TextDataItem), so I just replace instance.extract() with instance.split() as a workaround. A better solution would be to define a DataItem object with relevant extract() method for 'str'.
3. I cannot fix L75, so I commented out L71-L79.
4. I add some print() statements (commented out in the file)
5. For me, this test file is useful because it shows how to build graph based on texts.

**Comment by hugochan** [comment]
> @hugochan thanks for your reply. Here's the description for my second commit:
> 
> 1. fix import path.
> 2. I mock a vocab_util.py with only 2 modifications compare to '...modules.utils.vocab_utils' (You can easily search them with comment "Modification"). The reason is in our test example, 'str' has no extract() method specified (which would be defined in a class like Text2TextDataItem), so I just replace instance.extract() with instance.split() as a workaround. A better solution would be to define a DataItem object with relevant extract() method for 'str'.
> 3. I cannot fix L75, so I commented out L71-L79.
> 4. I add some print() statements (commented out in the file)
> 5. For me, this test file is useful because it shows how to build graph based on texts.

@WuXuan374 Thanks again for working on making our testing better! A few comments regarding the new changes:
2) I agreed that "A better solution would be to define a DataItem object with relevant extract() method for 'str'" so that we can directly call the built-in VocabModel. Can you please take this way?
3) Regarding L75, can you take a look into this [example](https://github.com/graph4ai/graph4nlp/blob/master/examples/pytorch/text_classification/run_text_classifier.py#L58) for reference?

Thanks for your time and contribution!

**Comment by WuXuan374** [comment]
@hugochan yes I will finish 2) and 3) in a few days.

**Comment by WuXuan374** [comment]
@hugochan Hi, I've fixed the above two problems in the newest commit.

File: graph4nlp/pytorch/test/graph_construction/run_batch_graph_embedding_construction.py [hunk 1]
```diff
@@ -3,34 +3,52 @@
...
+
+class RawTextDataItem(DataItem):
+    def __init__(self, input_text, tokenizer=None):
+        """
+        input_text: str
+        """
+        super(RawTextDataItem, self).__init__(input_text, tokenizer)
+    
+    def extract(self, lower_case=True):
+        return self.input_text.lower().split()
```
Comment (hugochan) [inline graph4nlp/pytorch/test/graph_construction/run_batch_graph_embedding_construction.py:22]: @WuXuan374 `lower_case` is not used in `extract` method.

File: graph4nlp/pytorch/test/graph_construction/run_batch_graph_embedding_construction.py [hunk 2]
```diff
@@ -3,34 +3,52 @@
 import numpy as np
 import torch
 
-from ...data.data import GraphData
-from ...modules.graph_construction.embedding_construction import EmbeddingConstruction
+from ...data.data import GraphData, to_batch
+from ...data.dataset import DataItem
+from ...modules.graph_embedding_initialization.embedding_construction import EmbeddingConstruction
```
Comment (hugochan) [inline graph4nlp/pytorch/test/graph_construction/run_batch_graph_embedding_construction.py:8]: @WuXuan374 Unused import path `EmbeddingConstruction`, please remove it.

File: graph4nlp/pytorch/test/graph_construction/run_batch_graph_embedding_construction.py [hunk 3]
```diff
@@ -41,34 +59,44 @@
...
             graph.node_attributes[j][
-                "token_idx"
-            ] = vocab_model.word_vocab.to_index_sequence_for_list(tokens)
-            node_size.append(len(graph.node_attributes[j]["token_idx"]))
+                "token_id"
+            ] = vocab_model.in_word_vocab.to_index_sequence_for_list(tokens)
+            node_size.append(len(graph.node_attributes[j]["token_id"]))
             max_node_len = max(node_size[-1], max_node_len)
 
+            # print('tokens: {}'.format(tokens))
```
Comment (hugochan) [inline graph4nlp/pytorch/test/graph_construction/run_batch_graph_embedding_construction.py:70]: @WuXuan374 can you remove L70~L74 comments?

File: graph4nlp/pytorch/test/graph_construction/run_batch_graph_embedding_construction.py [hunk 4]
```diff
@@ -41,34 +59,44 @@
...
-        tmp_token_idx = [graph.node_attributes[j]["token_idx"] for j in range(graph.get_node_num())]
+        tmp_token_idx = [graph.node_attributes[j]["token_id"] for j in range(graph.get_node_num())]
         tmp_token_idx = torch.LongTensor(
             (pad_2d_vals(tmp_token_idx, len(tmp_token_idx), max_node_len))
         )
-        graph.node_features["token_idx"] = tmp_token_idx
 
-    graph_list = [graph.to_dgl() for graph in graph_list]
-    bg = dgl.batch(graph_list, edge_attrs=None)
+        # print('after padding, token_id: {} {}'.format(tmp_token_idx.shape, tmp_token_idx))
```
Comment (hugochan) [inline graph4nlp/pytorch/test/graph_construction/run_batch_graph_embedding_construction.py:84]: @WuXuan374 can you remove this comment as well?

File: graph4nlp/pytorch/test/graph_construction/run_batch_graph_embedding_construction.py [hunk 5]
```diff
@@ -3,34 +3,52 @@
 import numpy as np
```
Comment (hugochan) [inline graph4nlp/pytorch/test/graph_construction/run_batch_graph_embedding_construction.py:3]: @WuXuan374 can you remove unused import `dgl`?

**Review by hugochan — CHANGES_REQUESTED** [review]
@WuXuan374 Looks great to me! Thank you for your contribution! Just left a few comments. You might also want to address the following lint warnings:
```
./graph4nlp/pytorch/test/graph_construction/run_batch_graph_embedding_construction.py:11:101: E501 line too long (113 > 100 characters)
./graph4nlp/pytorch/test/graph_construction/run_batch_graph_embedding_construction.py:20:1: W293 blank line contains whitespace
./graph4nlp/pytorch/test/graph_construction/run_batch_graph_embedding_construction.py:24:1: E305 expected 2 blank lines after class or function definition, found 1
./graph4nlp/pytorch/test/graph_construction/run_batch_graph_embedding_construction.py:44:1: W293 blank line contains whitespace
./graph4nlp/pytorch/test/graph_construction/run_batch_graph_embedding_construction.py:91:1: W293 blank line contains whitespace
```

You can find the above lint warnings by running ` ./dev/ci_lint.sh`.

**Comment by WuXuan374** [comment]
@hugochan fix lint warnings and above issues in newest commit

**Comment by hugochan** [comment]
@WuXuan374 LGTM. Thanks for your contribution!

**Pull request merged**